Products/ZenUtils/controlplane/client.py

```diff
@@ -222,12 +226,57 @@ def queryServiceInstances(self, serviceId):
 
     def queryServiceStatus(self, serviceId):
         """
-        Returns a sequence of ServiceInstance objects.
+        Returns a sequence of ServiceStatus objects.
+        """
+        if not self._useV2:
+            try:
+                response = self._dorequest("/services/%s/status" % serviceId)
+                body = ''.join(response.readlines())
+                response.close()
+                decoded = ServiceStatusJsonDecoder().decode(body)
+            except urllib2.HTTPError as ex:
+                if ex.getcode() == 404:
+                    self._useV2 = True
```

Comment:
Let's set this in the constructor since we will need it for other missing/modified endpoints